Components/Navigation/navigation.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation</title>
+    <link rel="stylesheet" href="/docs.css">
+    <link rel="stylesheet" href="/Components/Navigation/navigation.css">
+    <script src="https://kit.fontawesome.com/2a3afcb5fd.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="header">
+        <i class="fas fa-fighter-jet header-icon"></i>
+        <h1 class="header-text"></i>QuickUI</h1>
+        <a class="header-link" class="header-link" href="/index.html">Home</a>
+    </header>
+    <nav>
+        <ul class="navigation">
+            <li class="nav-link"><a class="nav-anchor" href="/Components/installation.html">Installation</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Avatar/avatar.html">Avatar</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Alert/alert.html">Alert</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Badge/badge.html">Badges</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Button/button.html">Button</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Card/card.html">Card</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Image/image.html">Image</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Input/input.html">Input</a></li>
+            <li class="nav-link"><a class="nav-anchor active"
+                    href="/Components/Navigation/navigation.html">Navigation</a></li>
+        </ul>
+    </nav>
+    <section class="main-container">
+        <h1 class="main-heading">Navigation</h1>
+        <div class="separator"></div>
+        <h3 class="sub-heading">Simple Navigation</h3>
+
+        <div class="header-outerbox">
```

Comment:
use <Header>

Reply:
I used header tag twice that why it automatically converted in div IG cause in my vscode from line 40 to 45 I used header tag

![Screenshot (30)](https://user-images.githubusercontent.com/79801546/152832410-378eedab-ef02-4eb6-9557-0526f35f795d.png)